docs/content/usage/issue-pull-request-templates.en-us.md

```diff
@@ -259,24 +276,29 @@ Validations:
 
 Attributes:
 
 | Key         | Description                                                                                           | Required | Type   | Default      | Valid values |
-|-------------|-------------------------------------------------------------------------------------------------------|----------|--------|--------------|--------------|
+| ----------- | ----------------------------------------------------------------------------------------------------- | -------- | ------ | ------------ | ------------ |
 | label       | A brief description of the expected user input, which is displayed in the form.                       | Required | String | -            | -            |
 | description | A description of the set of checkboxes, which is displayed in the form. Supports Markdown formatting. | Optional | String | Empty String | -            |
 | options     | An array of checkboxes that the user can select. For syntax, see below.                               | Required | Array  | -            | -            |
 
 For each value in the options array, you can set the following keys.
 
-| Key      | Description                                                                                                                              | Required | Type    | Default | Options |
-|----------|------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|---------|---------|
-| label    | The identifier for the option, which is displayed in the form. Markdown is supported for bold or italic text formatting, and hyperlinks. | Required | String  | -       | -       |
-| required | Prevents form submission until element is completed.                                                                                     | Optional | Boolean | false   | -       |
+| Key          | Description                                                                                                                                    | Required | Type         | Default | Options |
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------|---------|---------|
+| label        | The identifier for the option, which is displayed in the form. Markdown is supported for bold or italic text formatting, and hyperlinks.       | Required | String       | -       | -       |
+| required     | Prevents form submission until element is completed.                                                                                           | Optional | Boolean      | false   | -       |
+| visible      | Set visible to a specific checkbox. Valid options are "form" to render in only in the form, "content" to have it in the created issue and both | Optional | String array | false   | -       |
+
+visible: Default is **[form, content]**
```

Comment:
What's the difference between `both` and `[form, content]`?
Shouldn't it be the same value, namely `[form, content]` as it is more maintainable?

Reply:
Okay, I see, the text is simply written confusingly.
If I'm already misreading the text, then so will others

Reply:
e726ce01d052e583ec2a24ed606edf4d77725316